R/build_tree.R

```diff
@@ -31,7 +31,7 @@ build_nodes <- function(x, root, title, settings) {
   r$attributes <- pull_attr(cur_tib, settings$attributes)
   r$link <- cur_tib[[settings$link]]
 
-  if (settings$type %in% "spacy" & !is.null(r$word) & length(r$word) > 0) {
+  if (settings$type %in% "spacyr" & !is.null(r$word) & length(r$word) > 0) {
```

Comment:
Eventually will probably have to change the default type of spacyr to text and look for text here in case we allow more types of text methods. OK as is for now!